libs/libc/netdb/lib_gaistrerror.c

```diff
@@ -105,5 +116,12 @@ FAR const char *gai_strerror(int errnum)
     }
   while (ndxlow <= ndxhi);
 #endif
-  return "Unknown error";
+#ifdef CONFIG_LIBC_GAISTRERROR_ERRNUM
+  if (snprintf(s_err, sizeof(s_err), STRERROR_UNKNOWN " %d", errnum) > 0)
```

Comment:
I think it is better to
```suggestion
  if (snprintf(s_err, sizeof(s_err), STRERROR_UNKNOWN " %d", errnum) < sizeof(s_err))
```
as `-1` is unlikely to be returned with current usage, but overflow is better thing to catch

Reply:
Done.